### DISCUSSION TITLE
Avalonia.iOS support for .NET 8

### DISCUSSION THREAD

**Discussion by FabianGolembiewski-sp** [AvaloniaUI/Avalonia#13651] [General]
Will Avalonia.iOS get an update, to support .NET 8 in the next weeks?
Thanks and best regards,
Fabian

**Comment by maxkatz6** [comment]
Most platforms already support .NET 8.
For iOS you need either use nightly builds or update this package which was broken https://www.nuget.org/packages/HarfBuzzSharp.NativeAssets.iOS/2.8.2.5

**Comment by FabianGolembiewski-sp** [comment]
Ok, than I have to use nightly builds, but the latest Avalonia.iOS build of today is still on NET 7.0:
https://nuget-feed-nightly.avaloniaui.net/packages/avalonia.ios/11.1.999-cibuild0042021-beta

My current decision is recgarding the question, if I have to go with .NET 7 or wait maybe one or two weeks until next release/build and than could use .NET 8.

Thanks

**Reply by timunie** [reply]
you should be able to use nightly with NET8 on your side. Should just work afaik

**Reply by FabianGolembiewski-sp** [reply]
Nope, latest build Avalonia.IOS 11.1.999-cibuild0042104-beta is not compatible with net8.0

**Reply by maxkatz6** [reply]
It is. The only incompatibility is with HarfBuzz library, which was updated to the latest. And also tested with .NET 8.

**Comment by FabianGolembiewski-sp** [comment]
I'm a little bit confused, can i change the package target?
Visual Studio says it's incompatible, I should use .NET 7

![image](https://github.com/AvaloniaUI/Avalonia/assets/68863610/f3df6d14-778a-489d-abd6-14792d5a7e75)

Maybe I'm too foolish for it, sorry guys :-(

**Reply by maxkatz6** [reply]
It shows the **minimum** version of the .NET target. You can use Avalonia.iOS with any ios .NET target of 7 and **higher**.

**Reply by FabianGolembiewski-sp** [reply]
Hey, I got it running with .NET 8. The problems I had resided to Visual Studio 2022 with Mac and the runtimes.
After reinstall of sdks the problem is gone.
Thank you for your help, Avalonia is awesome :-)
